keys/phobos.cam.corp.google.com.crt.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This is useless to anyone not on my desktop, so will remove it and the associated key.